### ISSUE TITLE
Cheese Charger - C140478, low stock

### ISSUE THREAD

**Issue by jimmerson17** [Sorakage033/SlimeVR-CheeseCake#19]
Hi, is there another fuse that could be used in place of C140478?  It's currently low stock at JLCPCB.  Can C182445 be used instead?

**Comment by Sorakage033** [comment]
Yes, you can use C182445 or C14165 instead. And you can also use 0Ohm resister C17888 instead of using fuse 'cause most of charger don't output over 3A in 5V condition.
